spec/spec.go

```diff
@@ -49,6 +49,14 @@ type Fixtures struct {
 // HTTPVerb is a type for an HTTP verb like GET, POST, etc.
 type HTTPVerb string
 
+// Info is the `info` portion of an OpenAPI specification that contains meta
+// information about it.
+type Info struct {
```

Comment:
I wanted to call this `SpecInfo` because it's more description, but Golint tells me that the import of `spec.SpecInfo` "stutters" in that there's a repetition of "spec". It's not that big of a deal, so I just called it `Info` instead.